### DISCUSSION TITLE
How to rename alert box text from orchadcore cms to some xyz

### DISCUSSION THREAD

**Discussion by syedshadab19** [OrchardCMS/OrchardCore#12123] [Q&A]
Hi  I wanted below snapshot text to rename walert  "welcome to orchadcore cms, your site has been successfully created."  to "welcome to xyz , your site has been successfully created."
 
![image](https://user-images.githubusercontent.com/25971751/182022459-6021e520-1a74-4e1d-bf7d-941bc2fa6a14.png)

**Comment by ns8482e** [comment]
Enable Dashboard feature that should allow you to add/change widgets on Dashboard

**Reply by hishamco** [reply]
You could override `AdminDashboard` shape

**Reply by syedshadab19** [reply]
Thanks for the reply can you please specify the steps on how to override AdminDashboadShare @hishamco

**Reply by syedshadab19** [reply]
@ns8482e  thanks for the reply ill check it out

**Reply by hishamco** [reply]
You could override them in a new theme or add the .cshtml in views folder using the shape name. Please check our docs for overriding the views

https://docs.orchardcore.net/en/latest/docs/reference/modules/Templates/#overriding-views
